.cspell.json

```diff
@@ -1315,6 +1348,7 @@
     "traj",
     "trajs",
     "tridiagonal",
+    "ttyusb",
```

Comment:
Thank you for your recommendation and I'll remove the "ttyusb" because [your suggestion was merged](https://github.com/autowarefoundation/open-ad-kit-docs/pull/13/files#r915656348).